dashboard/consts.py

```diff
@@ -4,6 +4,11 @@
 DASHBOARD_AGENT_PORT_PREFIX = "DASHBOARD_AGENT_PORT_PREFIX:"
 DASHBOARD_AGENT_LOG_FILENAME = "dashboard_agent.log"
 DASHBOARD_AGENT_CHECK_PARENT_INTERVAL_SECONDS = 2
+RAY_STATE_SERVER_MAX_HTTP_REQUEST_ENV_NAME = "RAY_STATE_SERVER_MAX_HTTP_REQUEST"
+RAY_STATE_SERVER_MAX_HTTP_REQUEST = env_integer(
+    RAY_STATE_SERVER_MAX_HTTP_REQUEST_ENV_NAME, 100
+)
+RAY_STATE_SERVER_MAX_HTTP_REQUEST_ALLOWED = 1000
```

Comment:
What's the difference between `RAY_STATE_SERVER_MAX_HTTP_REQUEST ` and `RAY_STATE_SERVER_MAX_HTTP_REQUEST_ALLOWED `? Can you also add comments to explain variables?